bin/mail/2016-11-15-hoc-global-minecraft/send-real-emails

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+# 750550 teachers
+../send-to-mailing-list --limit 751000 2016-11-15-hoc-global-minecraft ./teachers.csv
```

Comment:
I always run the query again before sending, and check the new total. If it's > than the limit in the PR I modify it locally before sending so we don't miss any.